src/router.js

```diff
@@ -194,6 +194,7 @@ reused after a route change, but that shouldn't happen given our use of the
           // used before the navigation is confirmed, for example, to update the
           // DOM. Given that, we clear the data immediately.
           resource.data = null;
+          router.currentRoute.value.meta.standalone = false;
```

Comment:
Why is this needed? Won't `router.currentRoute.value` change after the next line, which ultimately navigates to either / or /login? Both of those routes have `standalone: false`, so I feel like this change to the value of `meta.standalone` should happen automatically. I'm sure I'm missing some aspect of this though.